src/js/utils/JestUtil.js

```diff
@@ -88,6 +140,46 @@ const JestUtil = {
   },
 
   /**
+   * Generates a callback function to a reduce() call that will
+   * find and return the textContent of DOM element(s) matching
+   * the given selector
+   *
+   * @example <caption>How to use reduceTextContentOfSelector</caption>
+   * const TestUtils = require('react-addons-test-utils');
+   *
+   * var componentInstance = TestUtils.renderIntoDocument(
+   *   <MyTableComponent />
+   * );
+   *
+   * var names = TestUtils.scryRenderedDOMComponentsWithClass(
+   *   componentInstance,
+   *   'table-column-name'
+   * ).reduce(
+   *   reduceTextContentOfSelector('.collapsing-string-full-string'),
+   *   []
+   * );
+   *
+   * @param {string} selector - The CSS selector to match children
+   * @returns {function} Returns a callback function to be passed on .reduce()
+   */
+  reduceTextContentOfSelector(selector) {
+    return function (strings, element) {
+      let matchedElements = Array.prototype.slice.call(
+          element.querySelectorAll(selector)
```

Comment:
If it doesn't find anything it should return an empty NodeList:
https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelectorAll#Notes

So this should work:
![](https://cl.ly/2c1R130j311G/Image%202016-09-16%20at%2014.26.34.png)

If not (and this might be a better solution), let's do this:
```js
matchedElements = Array.from(element.querySelectorAll(selector));
matchedElements.forEach(function (stringElement) { ... });
```